### DISCUSSION TITLE
Config pysumma Locally

### DISCUSSION THREAD

**Discussion by IanWhidden** [UW-Hydro/pysumma#168] [General]
Does the suggested install through conda work best on Mac OS or Windows OS?
When installing on a Mac OS, what is the role of docker when installing if I am running the command line through terminal on Mac OS?
I've tried the install on my Mac OS using the conda, not pip, and have received the bellow errors. 

<img width="427" alt="ErrorRef1" src="https://user-images.githubusercontent.com/103448529/193470405-e0b275af-7032-47e4-81b6-378f906fd3f2.png">
<img width="468" alt="ErrorRef2" src="https://user-images.githubusercontent.com/103448529/193470407-b6a33a3f-7393-4235-bd50-d9841fde5e3f.png">

**Comment by arbennett** [comment]
pysumma (and SUMMA more broadly) is not supported on Windows. Both MacOS or Linux should work out of the box with the conda installs though.

Docker is not directly needed for any of the SUMMA/pysumma functionality, although there is a Docker build of SUMMA that can be used. But I think that approach has fallen out of favor, and the conda builds are preferable. Or alternatively building SUMMA from source and then using that build to connect to pysumma.

I'm not really sure what would have caused the messages you are seeing either. Can you report what you ran as the command? And if possible a full copy-and-paste of the entire error message to help debug?

**Reply by IanWhidden** [reply]
Hi Andrew,

I ended up successfully installing pySUMMA, through pip, and creating a python environment. I had SUMMA already installed on my Mac OS, so installing through pip ended up working well.

I gave up on the Windows OS attempt.

In the future I will copy and paste the entire error message.

Thank you.
